src/coreclr/tools/Common/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -184,9 +184,13 @@ public static unsafe PEReader OpenPEFile(string filePath, out MemoryMappedViewAc
 
         private EcmaModule AddModule(string filePath, string expectedSimpleName, bool useForBinding, ModuleData oldModuleData = null, bool throwOnFailureToLoad = true)
         {
-            filePath = Path.GetFullPath(filePath);
+            if (filePath != null)
```

Comment:
It would be a simpler change to move the `filePath = Path.GetFullPath(filePath);` line right before `peReader = OpenPEFile(filePath, out mappedViewAccessor);`. At that point `filePath` cannot be null.

Reply:
Thank you. Fixed.